hw1/02-palindrome.js

```diff
@@ -1,3 +1,17 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+function handleInput() {
+  let rev = elem.value.split("").reverse().join("");
+  if (elem.value < 0 || elem.value == NaN) {
+    document.getElementById("error").style.display = "block";
```

Comment:
Since you are querying for the result and error element multiple times in your code, you can initialize the elements outside the function and access it so that the querying is done only once instead of multiple times (whenever user changes input). I believe this would optimize your program.